packages/block-editor/src/components/height-control/index.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * WordPress dependencies
+ */
+import { useMemo } from '@wordpress/element';
+import {
+	BaseControl,
+	RangeControl,
+	Flex,
+	FlexItem,
+	__experimentalSpacer as Spacer,
+	__experimentalUseCustomUnits as useCustomUnits,
+	__experimentalUnitControl as UnitControl,
+	__experimentalParseQuantityAndUnitFromRawValue as parseQuantityAndUnitFromRawValue,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import useSetting from '../use-setting';
+
+const CUSTOM_VALUE_SETTINGS = {
+	px: { max: 1000, steps: 1 },
+	'%': { max: 100, steps: 1 },
+	vw: { max: 100, steps: 1 },
+	vh: { max: 100, steps: 1 },
+	em: { max: 50, steps: 0.1 },
+	rm: { max: 50, steps: 0.1 },
```

Comment:
Is there anything against using `rem`? 😄 
```suggestion
	rem: { max: 50, steps: 0.1 },
```

Reply:
> Is there anything against using rem? 😄

Actually, we should be using `rem` otherwise the control won't pick up the max value and only use the default `10`

Reply:
Good catch, cheers! Looks like a typo from going too quickly at the end of the week 😄